src/common/components/select-repository-list/styles.css

```diff
@@ -7,9 +7,11 @@
 }
 
 .summary {
+  display: flex;
   flex: 1;
   margin-right: 10px;
   text-align: right;
+  justify-content: flex-end;
```

Comment:
I think this is not needed.

Text was right aligned just fine before. `justify-content: flex-end;` is needed because `display: flex;` was added here as well. If both `flex` and `justify-content` are removed `text-align` does the trick.